web/src/hooks/use-list-workflow-runs-api.ts

```diff
@@ -26,7 +28,7 @@ const REFETCH_INTERVAL_1_SECOND = 1_000; // in ms
 export const useListWorkflowRunsApi = (workflowId: string, limit?: number) => {
 	return useQuery({
 		queryKey: ["workflowRuns", workflowId, limit],
-		queryFn: () => buildListWorkflowRunsApi(workflowId, limit)(),
+		queryFn: () => buildListWorkflowRunsApi(workflowId, limit)({}),
 		refetchInterval: REFETCH_INTERVAL_1_SECOND,
```

Comment:
Why are you passing `limit` to `buildListWorkflowRunsApi` and not to the body? Right now, you are not using `limit` anywhere.

It should be:
```
const buildListWorkflowRunsApi = (workflowId: string) =>
```
and
```
		queryFn: () => buildListWorkflowRunsApi(workflowId)({ limit }),
```